alg-excer4.js

```diff
@@ -0,0 +1,51 @@
+/*Create a function that moves all capital letters to the front of a word.
+Examples
+capToFront("hApPy") ➞ "APhpy"
+Assumptions:
+* No spaces*/
+
+const moveCapLetter = str => {
+  var capLetters = "";
+  var smallLetter = "";
+  var updatedstr = "";
+
+  for (var i = 0; i < str.length; i++) {
+    if (str.charCodeAt(i) >= 65 && str.charCodeAt(i) <= 90) {
+      capLetters += str[i];
+    } else {
+      smallLetter += str[i];
+    }
+  }
+  updatedstr = concatStr(capLetters, smallLetter);
```

Comment:
Nice job,

For this you don't need to use the charCodeAt but it does give you the idea the ranges.

If there are no other characters besides alphabets then we can just check for  `str[i] <= "Z" ` or `str[i] < "a" `